actionview/lib/action_view/railtie.rb

```diff
@@ -51,6 +51,14 @@ class Railtie < Rails::Engine # :nodoc:
       end
     end
 
+    initializer "action_view.setup_etag_with_template_digest" do |app|
+      ActiveSupport.on_load(:action_controller_base) do
+        etag do |options|
```

Comment:
You might want to make this conditional with `if self < EtagWithTemplateDigest`, in case someone excludes it with `without_modules`.

Once we're doing that, we may as well add a action_controller_api` hook that does the same thing so that if someone happens to use this with ActionController::API, it will work out of the box.  If not, we at least need to document someone that this is a breaking change, as simply including the module in ActionController::API (or in any class for that matter) is no longer enough to make it work.